### DISCUSSION TITLE
[Request] Discombobulator button

### DISCUSSION THREAD

**Discussion by JohnMetroid** [Malcolm-Q/LC-LateGameUpgrades#409] [General]
Our group has been playing with LGU for a little while now, and the one thing we keep missing is the ability to activate the discombobulator a bit more easily.

It would be great if there was a button added when you unlocked the upgrade, similar to teleport/ITP

Thanks for the consideration

![image](https://github.com/Malcolm-Q/LC-LateGameUpgrades/assets/139426529/b85c5584-19e9-4911-bd88-0214795b3f9c)

**Comment by JohnMetroid** [comment]
As a minor update, the base behavior of discombobulator is something my group has grown to dislike so much that we just went ahead and disabled it.  As lethal modding scales up, using the discombobulator in the vanilla fashion becomes even more cumbersome/unwieldy.

We would continue to use this upgrade if it were integrated more practically.

Thanks again for the consideration.

**Comment by maximo4** [comment]
I think it's an excellent suggestion. Personally, I use it, but I'm the only one doing so because others are unaware of it, despite the time I take to explain and show the important features of the modpack, which still isn't enough.

**Comment by JohnMetroid** [comment]
I believe the last LGU update actually addressed this indirectly.  All that's left is someone with the time/interest to make a standalone mod that hooks into the discombobulator API, assuming everything is functional at the moment

![image](https://github.com/user-attachments/assets/2a721a36-0aec-45ff-b2cf-3ac6443f4509)

**Reply by Shaydelity** [reply]
ScienceBird_Tweaks adds buttons that could use it, given this the Discombulator API ist made to be compatible
